inc/core.php

```diff
@@ -55,3 +55,12 @@ function get_package_path($id, $version) {
 	// This is safe - These values have been validated via validateIdAndVersion above
 	return '/packagefiles/' . $id . '/' . $version . '.nupkg';
 }
+
+/* Used to construct URIs */
+function http_https() {
```

Comment:
Sure :)

btw you don't need to manually squash your commits, I can do it automatically in the Github UI when merging the pull request.